### ISSUE TITLE
remove "first up: Truth" screen?

### ISSUE THREAD

**Issue by cosmicespresso** [Kallirroi/bot_test#82]
kind of don't want to have to change all the numbering again but maybe it doesn't make so much sense to have that?

**Comment by agnescameron** [comment]
yeah i don't mind it but i know what you mean... though perhaps it is useful in steering the conversation? 

don't feel strongly either way!

**Comment by cosmicespresso** [comment]
What I think is a bit off is that the word "truth" appears in both this screen as well as the one before - maybe just remove one of the two occurrences?

**Comment by agnescameron** [comment]
oh yeah that's a point... we could even be a bit more instructional about it? Like: 'it's your turn to answer a challenge'
